template/.vscode/settings.json

```diff
@@ -26,5 +26,8 @@
   },
   "[typescriptreact]": {
     "editor.formatOnSave": false
```

Comment:
do we still want to set this to false?

Reply:
I'd vote for true, this is super handy

Reply:
Dang it that is what it should have been. It's updating to the new prop and I copy pasta wrong.

Reply:
I will get this in the next PR I put up, I know there will be more to update.